app/Http/Controllers/AdminConabController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Validator;
+
+class AdminConabController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $user = Auth::user();
+        $users = User::where('user_type', 'ADMIN_CONAB')->where('id', '<>', $user->id)->get();
+        return $users;
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
```

Comment:
Tem que levar em consideração que não é qualquer adiministrador que terá acesso a rota store. Seria uma boa você criar um atributo no banco que distinguisse admins conab com permissão de criar outros admins e validar isso antes de realizar o store. Retornar o atributo também seria uma boa pra que quando você pegasse o front, pudesse esconder essa funcionalidade na view e, caso queira, nos middlewares do component Route.

Reply:
Isso não está no requisito, se o usuário é administrado da conab ele pode cadastrar outros administradores.

Reply:
Certo, mas então qual o sentido que falar sobre CLI e administrador root então se não era necessário, lembro que você falou isso na reunião com Gustavo também e esse não foi um ponto citado.

Reply:
'se o usuário é administrado da conab ele pode cadastrar outros administradores.', apesar de eu não concordar totalmente com essa afirmação, como você disse, não ficou explicíto no requisito, então realmente ficar aberto pro que o cara que está desenvolvendo acha.

Reply:
Mas apesar de do jeito que o requisito está, o store fazer sentido devido a falta de informações sobre regra de negócio, o fato de um administrador poder alterar quaisquer dados de outro administrador que não seja o password não faz. Caso um cara mal intencionado queira, a aplicação vai estar permitindo que ele altera  email de outro administrador e impeça o cara de fazer login. E do jeito que está não há log pra saber quem foi.

Reply:
> Certo, mas então qual o sentido que falar sobre CLI e administrador root então se não era necessário, lembro que você falou isso na reunião com Gustavo também e esse não foi um ponto citado.

Do mesmo modo, não faz sentido ter um requisito implícito esqueceu senha, se qualquer admin pode alterar a senha do outro.

Reply:
A gente tinha decidido isso, no frontend não tem a opção de um administrador mudar os dados do outro. A senha que fiz foi baseada no cpf do administrador, ou seja, é como no suap a primeira senha todo mundo sabe qual é a sua (seu cpf e alguma coisa), aqui foi só o cpf. Sobre o superadmin como Gustavo falou na vdd um usuário fake só pra ver o fluxo do login e cadastro de administradores.

Reply:
Ae quando o novo administrador for entrar na aplicação ele deve alterar sua senha, como a gente faz no sua.

Reply:
Então, não estou discutindo sobrea a senha ser o cpf. Vou requisitar outra mudança porque essa thread com o que Gustavo falou já está resolvida.